units/systemd-timesyncd.service.in

```diff
@@ -24,8 +25,9 @@ RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-timesyncd
 WatchdogSec=3min
 CapabilityBoundingSet=CAP_SYS_TIME CAP_SETUID CAP_SETGID CAP_SETPCAP CAP_CHOWN CAP_DAC_OVERRIDE CAP_FOWNER
+DevicePolicy=closed
+DeviceAllow=/dev/rtc rw
```

Comment:
In light what I suggested above, this would then have to be changed to "DeviceAllow=char-rtc rw" or so, i.e. be a generic match against the whole subsystem, instead of a specific device that might never appear